DELIVERY.README.txt

```diff
@@ -133,12 +131,8 @@ Contains the final VCF files and indexes, in total 6 files.
 Genomic VCF (gVCF) (and index file) containing sequencing information for both
 variant and non-variant positions. Can be used for downstream cohort calling.
 
---SampleN.clean.dedup.recal.bam.recalibrated.indel.annotated.vcf.gz
---SampleN.clean.dedup.recal.bam.recalibrated.indel.annotated.vcf.gz.tbi
-VCF file containing indel variants called from the recalibrated BAM file and
+--SampleN.clean.dedup.recal.bam.raw.annotated.vcf.gz
+--SampleN.clean.dedup.recal.bam.raw.annotated.vcf.gz.tbi
+VCF file containing variants called from the recalibrated BAM file and
```

Comment:
snps and indels have been always separated but this has been a design choice from @johandahlberg  . I really do not know the reason but it should not be a problem.